redaxo/src/addons/users/pages/users.php

```diff
@@ -149,7 +151,9 @@
     }
 
     if ('' != $userpsw) {
-        $updateuser->setValue('password', rex_login::passwordHash($userpsw));
+        $updateuser->setValue('password', $passwordHash = rex_login::passwordHash($userpsw));
+        $updateuser->setValue('password_changed', time());
+        $updateuser->setArrayValue('previous_passwords', $passwordPolicy->updatePreviousPasswords($user, $passwordHash));
```

Comment:
bei mir steht nur ein leeres Array in der Datenbank `[]` ; Auch nach mehrfachen ändern bleibt dies so.

Reply:
ah ok, wenn `not_last` und `not_months` net gesetzt sind ist das scheinbar so.

Reply:
Genau, die werden immer entsprechend der Einstellung aufgeräumt.